projboard/tests/test_article.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from projboard.models.article import Article, Like, View, Subject, User
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+class TestArticle:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_supjects(self):
```

Comment:
please change to `create_subjects(self):`

Reply:
> please change to `create_subjects(self):`

@Anas-mhameed ^^